pool/connection_pool.go

```diff
@@ -913,9 +896,6 @@ func (p *ConnectionPool) NewPrepared(expr string, userMode Mode) (*tarantool.Pre
 
 // NewWatcher creates a new Watcher object for the connection pool.
 //
-// You need to require IPROTO_FEATURE_WATCHERS to use watchers, see examples
-// for the function.
-//
 // The behavior is same as if Connection.NewWatcher() called for each
 // connection with a suitable role.
```

Comment:
We also add a description of the behavior:

A watcher could be created only for instances with the support.
```suggestion
// The behavior is same as if Connection.NewWatcher() called for each
// connection with a suitable mode.
```